cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -1,4 +1,5 @@
 import attr
+from cached_property import cached_property
```

Comment:
I think better to group them like:
import attr
import fauxfactory
from cached_property import cached_property
This is not in alphabetic but I think will look better is we also take into account kind of import